client/authorization/components/signInDialog.js

```diff
@@ -4,22 +4,60 @@ import {connect} from 'react-redux';
 import * as actions from '../actions';
 import Popup from './popup';
 import SignInForm from './signInForm';
+import SignUpForm from './signUpForm';
 import classNames from './scss/button.module.scss';
+import TabBar from '../../components/tabBar';
+import classNames1 from './scss/signInForm.module.scss'
```

Comment:
why do you need two files with class names? Probably you need to refactor that to follow BEM